app/components/em-list.js

```diff
@@ -0,0 +1,75 @@
+import Ember from 'ember';
+import { task, timeout } from 'ember-concurrency';
+import filterByQuery from 'ember-cli-filter-by-query/util/filter';
+
+const {
+  Component,
+  computed
+} = Ember;
+
+const QUERY_DEBOUNCE = 200;
+
+export default Component.extend({
+  tagName: 'table',
+  classNames: 'table table-packages',
+
+  init() {
+    this._super(...arguments);
+    this._filterList(this.get('query'));
+    Ember.run.scheduleOnce('afterRender', () => {
```

Comment:
ScheduleOnce with an anonymous function won't work, as each invocation will get a new function so the unique check will fail.

In this scenario, init is already only invoked once per component so the once thing has no meaning, it only costs more than then equivalent schedule